backend/app2/llm/music_gen_service/midi2.py

```
@@ -0,0 +1,374 @@
+    Returns:
+        List of MIDI note values
+    """
+    logger.info(f"Parsing chord name: {chord_name}")
+    # Convert 'b' flats to '-' for music21
+    chord_name = chord_name.replace("b", "-")
+    chord = harmony.ChordSymbol(chord_name)
+    return [note.midi for note in chord.pitches]
+
+
+def _convert_duration_to_beats(
+    duration_str, time_signature: List[int] = [4, 4]
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)